### PR TITLE
store video on the backend

### PR DESCRIPTION
**Pull request by asood123**

### CODE REVIEW

File: test/server/banner.spec.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+import sinon from 'sinon';
+import sizeOf from 'image-size';
 
 import api from '../../server/src/lib/api';
 import app from '../../server/src/index';
-import request from 'supertest';
-import sizeOf from 'image-size';
 
-import sinon from 'sinon';
+import shieldIONock from '../data/shields.io.nock'; // eslint-disable-line
```
Comment (sedubois) [inline test/server/banner.spec.js]: why is the disable-line necessary?

File: test/server/banner.spec.js [hunk 2]
```diff
@@ -89,6 +90,8 @@ describe("badge", () => {
     });
   });
 
+  beforeEach(() => shieldIONock());
```
Comment (sedubois) [inline test/server/banner.spec.js]: can also write `beforeEach(shieldIONock);`

File: test/server/banner.spec.js (hunk 1 above)
Comment (asood123) [inline test/server/banner.spec.js]: not sure why i added that. removed.

File: test/server/banner.spec.js (hunk 2 above)
Comment (asood123) [inline test/server/banner.spec.js]: changed.

**Pull request merged**